code/modules/economy/commodity.dm

```diff
@@ -2900,3 +2880,1072 @@
+		reagents.add_reagent("toxic_slurry", 20)
+		reagents.add_reagent("water", 10)
+/obj/item/reagent_containers/glass/bottle/plasma
+	name = "Bottle of Liquid Plasma"
+	desc = "A bottle with a label on it. The label says 'PLASMA'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("liquid plasma", 50)
+/obj/item/reagent_containers/glass/flask/blobfood
+	name = "Flask of Nutrient Sludge"
+	desc = "A bottle with a label on it. The label says 'NUTRITION MIX #4'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("enriched_msg", 5)
+		reagents.add_reagent("tonic water", 5)
+		reagents.add_reagent("beff", 5)
+		reagents.add_reagent("juice_grapefruit", 10)
+		reagents.add_reagent("water", 10)
+/obj/item/reagent_containers/glass/flask/gravy
+	name = "Flask of Gravy"
+	desc = "A bottle with a label on it. The label says 'GRAVY'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("gravy", 45,"msg", 5)
+/obj/item/reagent_containers/glass/flask/milk
+	name = "Flask of Milk"
+	desc = "A flask with a label on it. The label says 'MILK'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("milk", 50)
+/obj/item/reagent_containers/glass/flask/insulin
+	name = "Flask of Insulin"
+	desc = "A flask with a label on it. The label says 'INSULIN'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("insulin", 50)
+/obj/item/reagent_containers/glass/flask/synthflesh
+	name = "Flask of Synthetic Flesh"
+	desc = "A dirty flask with a label on it. The label says 'SYNTHETIC FLESH'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("synthflesh", 50)
+/obj/item/reagent_containers/glass/flask/triplepiss
+	name = "Flask of Triple Piss"
+	desc = "A dirty flask with a label on it. The label says 'TRIPLEPISS'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("triplepiss", 50)
+/obj/item/reagent_containers/glass/flask/enzyme
+	name = "Flask of Booster Enzymes"
+	desc = "A dirty flask with a label on it. The label says 'BOOSTER ENZYME'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 20
+	New()
+		..()
+		reagents.add_reagent("enzymatic_leftovers", 5)
+		reagents.add_reagent("denatured_enzyme", 5)
+		reagents.add_reagent("booster_enzyme", 10)
+/obj/item/reagent_containers/glass/flask/oil
+	name = "Flask of Oil"
+	desc = "A flask with a label on it. The label says 'OIL'."
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 10
+	initial_volume = 50
+	New()
+		..()
+		reagents.add_reagent("triplepiss", 50)
```

Comment:
The oil flask seems to create triplepiss. I'm assuming you forgot to change it to oil when copypasting the triplepiss flask.